binder/plugins/my_filters/models.py

```diff
@@ -13,6 +13,7 @@ class MyFilter(BinderModel):
     view = models.TextField()
     name = models.TextField()
     params = models.JSONField()
+    columns = models.JSONField(default=list, blank=True)
```

Comment:
Can you add two backend tests to the test suite?

- one where we use my_filters to add actual filters and keep column empty
- one where we fill the column value.

At the moment, I think there are no tests, and thus, this could break if people start tweaking it in the future and rely only on test coverage.

If you have issues running test or need typing help just LMK.

Reply:
In my opinion, we should do the testing for this in spider & IAE.

Testing this in binder is very complicated, and not really worth it since it has almost 0 complexity in binder. I will review the corresponding spider & IAE PRs first.